### PR TITLE
SNAP-12099 May release announcements added to page

### PR DESCRIPTION
**Pull request by ddellsperger**
Make sure you've checked off all these things before submitting:

- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `main` branch.
- [x] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -292,7 +292,8 @@ Choose a number or apply filter (format: [groupId:]artifactId, case sensitive co
 Choose com.snaplogic.tools:SnapArchetype version: 
 1: 35.0
 2: 36.0
-Choose a number: 2:
+3: 37.0
```
Comment (rohithmadhavan) [inline source/index.html.md:295]: Can remove 35.0 from the archetype catalog.

**Review by rohithmadhavan — APPROVED** [review]

File: source/index.html.md (hunk 1 above)
Comment (ddellsperger) [reply source/index.html.md:295]: I guess this is a good question, do we want to remove versions to have only 2 versions available at a time? we previously didn't do that, but I cleared a large number of them out recently.

Comment (rohithmadhavan) [reply source/index.html.md:295]: Yes, we should only keep the active versions going forward.

Comment (ddellsperger) [reply source/index.html.md]: alright, removed 35.0 from the list in the log, I'll handle this when the Archetype items are updated, approved, and deployed.

**Review by pmeister — APPROVED** [review]

**Pull request merged**